runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/components/BasicTextField.tsx

```diff
@@ -64,19 +71,22 @@ const BasicTextField: FC<BasicTextFieldProps> = ({
     }, [debouncedValue]);
 
     return (
-        <TextField
-            {...params}
-            value={currentValue}
-            fullWidth
-            variant="outlined"
-            disabled={isFieldDisabled}
-            required={props.required}
-            label={props.label}
-            onChange={(event) => handleChange(event.target.value)}
-            InputLabelProps={{ shrink: true }}
-            error={Boolean(errors) || Boolean(props.rawErrors)}
-            helperText={errors ? errors : null}
-        />
+        <FieldWithTooltipWrapper>
+            <TextField
+                {...params}
+                value={currentValue}
+                fullWidth
+                variant="outlined"
+                disabled={isFieldDisabled}
+                required={props.required}
+                label={props.label}
+                onChange={(event) => handleChange(event.target.value)}
+                InputLabelProps={{ shrink: true }}
+                error={Boolean(errors) || Boolean(props.rawErrors)}
+                helperText={errors ? errors : null}
+            />
+            <InfoTooltip text={props.options?.info}/>
```

Comment:
this should be displayed conditionally - if there's no props.options?.info then there's no InfoTooltip. 
You can use <If /> component (If.tsx)

Reply:
Oh I see you've used that inside a component. It's shorter but imo we prefer:
`
<If condition={ Boolean(props.options?.info) } >
<InfoTooltip text={props.options.info}/>
</If>
`

because you instantly see that it's conditional component